packages/app/src/cli/services/bulk-operations/download-bulk-operation-results.ts

```diff
@@ -0,0 +1,12 @@
+import {fetch} from '@shopify/cli-kit/node/http'
+import {AbortError} from '@shopify/cli-kit/node/error'
+
+export async function downloadBulkOperationResults(url: string): Promise<string> {
+  const response = await fetch(url)
+
+  if (!response.ok) {
+    throw new AbortError(`Failed to download bulk operation results: ${response.statusText}`)
+  }
+
+  return response.text()
+}
```

Comment:
Why have this function in a different file? is just wrapping a call to `fetch`​ and a error? And is only used in a single place... best to put the call to fetch directly where it is needed. I'd say the tests don't add much value either (you are just testing an `if`​)